source/common/config/utility.cc

```diff
@@ -271,7 +271,7 @@ void Utility::translateOpaqueConfig(const ProtobufWkt::Any& typed_config,
   if (!typed_config.value().empty()) {
     // Unpack methods will only use the fully qualified type name after the last '/'.
     // https://github.com/protocolbuffers/protobuf/blob/3.6.x/src/google/protobuf/any.proto#L87
-    absl::string_view type = TypeUtil::typeUrlToDescriptorFullName(typed_config.type_url());
+    absl::string_view type = Config::TypeUtil::typeUrlToDescriptorFullName(typed_config.type_url());
```

Comment:
I think the conversion utils belong in `source/common/protobuf`, not `source/common/config`, since they are about type URLs, and hence generic to all protobuf.

Reply:
Thanks, will remove the Config namespace.